src/model/paste/__tests__/DraftPasteProcessor-test.js

```diff
@@ -127,18 +127,22 @@ describe('DraftPasteProcessor', function() {
     ]);
   });
 
-  /**
-   * todo: azelenskiy
-   * Changes to the mocked DOM appear to have broken this.
-   *
-   * it('must suppress blocks nested inside other blocks', function() {
-   *   var html = '<p><h2>Some text here</h2> more text here </p>';
-   *   var output = DraftPasteProcessor.processHTML(html, CUSTOM_BLOCK_MAP);
-   *   assertBlockTypes(output, [
-   *   'unstyled',
-   *   ]);
-   * });
-   */
+  it('must suppress blocks nested inside other blocks', function() {
+    var html = '<h2><p>Some text here</p> more text here </h2>';
```

Comment:
`h2` is not a valid child of a `p` tag. The original HTML was parsed so that the body tag, therefore, had three children rather than one.